openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -44,18 +46,24 @@ def count_data():
     # Built-in edX`s accounts have None in `last_login` field.
     # 30th day <= Current last login date <= Today
     activity_border = (datetime.datetime.now() - datetime.timedelta(days=olga_settings.get("ACTIVITY_PERIOD")))
-    active_students_amount = UserProfile.objects.all().exclude(
-        user__last_login=None).filter(user__last_login__gt=activity_border).count()
+    active_students_amount = UserProfile.objects.exclude(
+        Q(user__last_login=None) | Q(user__is_active=False)
+    ).filter(user__last_login__gt=activity_border).count()
+
+    # Aggregated students per country
+    students_per_country = UserProfile.objects.exclude(
+        Q(user__last_login=None) | Q(user__is_active=False)
+    ).filter(user__last_login__gt=activity_border).values('country'
```

Comment:
you can probably save ```UserProfile.objects.exclude(Q(user__last_login=None) | Q(user__is_active=False))``` to variable and then use it in calculation of active_students_amount and students_per_country. But it is as you decide, you can skip this comment for now.